_layouts/example.html

```diff
@@ -3,7 +3,7 @@
 ---
 {% include learnnav.html showmenu=true %}
 
-<div class="section lightest">
+<div class="section lightest dark:dark">
```

Comment:
To not break anything wanted to have full control which elements will go "dark" in dark mode rather than applying dark mode to all "section" elements.